ducc/unitfiles/cvmfs_ducc.service.in

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=DUCC Daemon Unpacker Containers in CVMFs
+After=network.target dockerd.service
+Requires=dockerd.service
+
+[Service]
+Environment="RECIPE_FILE=YourRecipeFile_UPDATE_THIS_VALUE.yaml"
```

Comment:
For both variables let's simplify the template, like "UPDATE-ME.yaml" and "UPDATE-ME" for the password